index.html

```diff
@@ -11,49 +11,205 @@
+                <button type="button" class="start-collaboration">Start collaboration</button>
             </div>
-        </div>
-
-    </header>
-
-    <section class="flex-container">
-        <h1 class="title_name">
-            Charlie
-            Giorgania
-        </h1>
-        <p class="about_description">
-            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
-            works.
-            If you like what you see and have a project you need coded, don’t hesitate and contact me.
-        </p>
-        <a><div class="collaborate_button">
-            <p id="start-collaboration" >start collaboration</p>
-        </div>
-        </a>
 
-    </section>
-    <section class="image-container">
-        <div id ="image_geometry_2-1"></div>
-    </section>
+        </section>
+        <section class="image-container">
+            <div id="image_geometry_2-1"></div>
+        </section>
 
-    <section class="project_container">
-        <div class="project_title_holder">
-           <h3 id="project-heading">Project</h3>        
-        </div>
+        <section class="project_container">
+            <div class="section_title_holder">
+                <h3>Projects</h3>
+            </div>
+            <div class="project-grid">
 
+                <div class="grid-item1">
+                    <div class="grid-project-image1"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>JavaScript</p>
+                            </div>
+                        </div>
+                        <button type="button" class="see-project-button">See this project &nbsp;<i
+                                class="fas fa-arrow-right fa-lg"></i></button>
+                    </div>
+                </div>
+                <div class="grid-item">
+                    <div class="grid-project-image2"></div>
+                    <div class="project-details">
+                        <h5>
+                            Project details goes here
+                        </h5>
+                        <div class="technology-grid">
+                            <div class="technology-grid-item">
+                                <p> HTML/CSS</p>
+                            </div>
+                            <div class="technology-grid-item">
+                                <p>Ruby on Rails</p>
+                            </div>
```

Comment:
- [ ] One of the requirements says that all projects in my work section should be in the `<ul>` tag like this:
```
 <ul class="project-container">
      <li class="project2"></li>
      <li class="project3"></li>
      <li class="project4"></li>
      <li class="project5"></li>
      <li class="project6"></li>
</ul>
``` 
- [ ] the work container should also use the `grid` CSS property.